meeting_minutes/221027_SARIF_TC_69.md

```diff
@@ -53,7 +53,6 @@ Chris Wysopal
 ### 1.7.5 Members who have declared a leave of absence
 
 Aditya Sharad
-Jeff Williams
```

Comment:
does he need to be moved to a different section within the minutes?

Reply:
I think not. As he only lost voting rights after this meeting (2022-NOV-10)

Reply:
Looking back at my records, I think I was supposed to remove his voting privileges at the previous meeting, but it doesn't matter because I have to do it anyway after today's meeting.

So, @katrinaoneil , I think you are right, but we don't have to be that precise this time because everything worked out the same in the end.

Reply:
:)